examples/72-gitingest-es/compose.dev.yml

```diff
@@ -0,0 +1,181 @@
+name: parakeet-genai-stack
+
+# Embeddings generation then start the application:
+# docker compose -f compose.dev.yml --profile generation --profile application up --build 
+
+# Start the application only:
+# docker compose -f compose.dev.yml --profile application up --build 
+
+services:
+
+  gitingest:
+    profiles: [generation]
+    build:
+      context: ./gitingest
+      dockerfile: Dockerfile
+    environment:
+      - GITHUB_REPOSITORY=https://github.com/parakeet-nest/parakeet
+      - INCLUDE_PATTERNS=*.go
+      #- INCLUDE_PATTERNS=*.go,*.md
+    volumes:
+      - ./data:/app/data
+
+  elasticsearch:
+    profiles: [generation, application]
+    image: docker.elastic.co/elasticsearch/elasticsearch:8.17.0
```

Comment:
nit maybe use latest? or you can use this which uses host networking so it drops in with any other docker compose without composition. https://github.com/elastic/elasticsearch-labs/tree/main/docker